otter/tap/api.py

```diff
@@ -303,7 +303,7 @@ def setup_converger(parent, kz_client, dispatcher, interval, build_timeout):
     Create a Converger service, which has a Partitioner as a child service, so
     that if the Converger is stopped, the partitioner is also stopped.
     """
-    converger_buckets = range(1, 10)
+    converger_buckets = range(10)
```

Comment:
Note to reviewer - also snuck in this change - I've filed issue https://github.com/rackerlabs/otter/issues/1493 to deal with it long term but this is a quick fix for now.